docs/dpdk/mlnx-dpdk/mlnx-pod_testpmd_connectx5.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   containers:
   - name: testpmd
     image: <DPDK testpmd image>
-    imagePullPolicy: Never
+    imagePullPolicy: IfNotPresent
     securityContext:
      capabilities:
         add: ["IPC_LOCK"]
```

Comment:
Dont we need CAP_NET_ADMIN (e.g for mtu changes) ?
Also i think we should explicitly state here that CAP_NET_RAW is desired if only as declarative statement in case the low-level runtime enables it by default.

Reply:
Currently I didn't see any cni that allow to configure MTU and link state in the container. for example OVN  configure mtu with  the cni. In sriov cni the link state is configurable by the cni. 
I do  think we need to add MTU support to sriov-cni.

Reply:
i see, so your take is to go with the minimal set of capabilities for DPDK container.